src/openforms/prefill/__init__.py

```diff
@@ -64,13 +64,19 @@ def invoke_plugin(item: Tuple[str, List[str]]) -> Tuple[str, Dict[str, Any]]:
 
         try:
             values = plugin.get_prefill_values(submission, fields)
+            if not values:
+                try:
+                    raise ValueError(f"Empty values for prefill plugin '{plugin_id}'")
+                except ValueError as e:
+                    logger.exception(f"exception in prefill plugin '{plugin_id}'")
+                    logevent.prefill_retrieve_failure(submission, plugin, e)
+                    return plugin_id, values
```

Comment:
this doesn't seem right - the fact that there's an empty values being returned doesn't mean/imply that there were errors/exceptions.

Instead below on line 72 I'd expect an `else` which logs a different message: `logevent.prefill_retrieve_empty` which explicitly mentions that "yes, the prefill API calls worked, but no data was returned" as that is the blind spot we currently have in logs